plenum/server/replica.py

```diff
@@ -534,14 +534,14 @@ def get_lowest_probable_prepared_certificate_in_view(
                 return n
         return None
 
-    def _setup_for_non_master(self):
+    def _setup_for_non_master(self, is_catchup=True):
```

Comment:
I think we need to
1) Rename this to `_setup_last_ordered_for_non_master`
2) Call it after catch-up ends, not only after view change ends.

Reply:
2. No, it's called only when catch-up is done as a part of view change. But it catchup is done because of, for example, outdated checkpoints, then it isn't called (the view and primary are still the same).